MTFHDataLanding/Helpers/Constants.cs

```diff
@@ -3,6 +3,7 @@ namespace MTFHDataLanding.Helpers
     public static class Constants
     {
         public const string BUCKET_NAME = "mtfh-data-landing-spike";
```

Comment:
can this be parameterised so that depending on the environment this is deployed to the bucket will change?

Reply:
The environment part is parameterized - the name is the same across all envs

Reply:
we probably don't want to include the fact that this is a spike? Are you intending to land data in our raw_zone?

Reply:
We can change the bucket name no problem, yes - whatever name we choose will be replicated on all our envs. What do you prefer?

Reply:
Hey, yeah so our bucket for landing data that requires anonymisation or is of a sensitive nature lands in `dataplatform-stg-landing-zone` & data that is not / or no longer requires anonymisation landas in `dataplatform-stg-raw-zone`. Are the data in these payloads of a sensitive nature?